FreeRTOS/FreeRTOS-Plus-TCP
# Flaten DNS files

Flatten DNS files

Description
-----------
Flatten DNS files by removing the DNS/ directory and merging all files in the parent directory.
change the buildsystem to point to the new location of files

Test Steps
-----------
Test cases build and pass


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
